tutorindigo/templates/indigo/cms/static/sass/partials/cms/theme/_variables.scss

```diff
@@ -0,0 +1,16 @@
+$primary:    #109dc2;
+$footer-bg:  #333;
+$blue:         $primary;
+
```

Comment:
Hi @Talha-Rizwan @Danyal-Faheem , We need to remove these `colors` variable and `fonts` variables as the work on styling the studio is in progress. Can you please do this so that I can merge it then.